.prettierignore

```diff
@@ -2,3 +2,7 @@ build/
 
 # Ignore markdown files till full pass is made on each folder
 *.md
+
+# The following paths have had a full pass
+!/files/zh-tw/games/**/*.md
+!/files/zh-tw/glossary/**/*.md
```

Comment:
Upstream, we've switched to using .prettierignore as a breakdown list rather than telling it to ignore everything and then un-ignoring specific folders.  It might be nice to do so here as well!

Reply:
I've made the changes in f58ed696ac11bdfb829d68696a5a6cd3113b83bb, could you have a double check on this :)